stories/button/button.stories.js

```diff
@@ -16,19 +16,19 @@ export default {
 
 export const primary = () => `
     <button class="fd-button">Create</button>
-    <button class="fd-button sap-icon--cart"></button>
+    <button class="fd-button"><i class="sap-icon--cart fd-button__icon-left"></i></button>
```

Comment:
This would be a good opportunity to add an `aria-label` to all the `<button>` elements with no inner text (similar to what is done in line 173).

Reply:
Done